lib/rex/random_identifier/generator.rb

```diff
@@ -111,11 +112,19 @@ class ExhaustedSpaceError < StandardError; end
     ).freeze
   )
 
+  PHPOpts = DefaultOpts.merge(
+    prefix: '$',
+    first_char_set: Rex::Text::Alpha + '_'
+    # nothing seems to be forbidden because everything is prefixed with '$'
+    # see: https://www.php.net/manual/en/reserved.php
```

Comment:
You'll probably want to avoid PHP's predefined variables https://www.php.net/manual/en/reserved.variables.php and the `$this` pseudo-variable etc